### DISCUSSION TITLE
Ormlite and EF Core

### DISCUSSION THREAD

**Discussion by tanczosm** [ServiceStack/Discuss#94] [Q&A] [answered]
In the new SS release if I migrate my AppUsers table over to EF Core as suggested can I still continue to access it with OrmLite?  Does OrmLite just intuitively figure out how to map columns from AspNetUsers to the AppUser non-snake case properties (assuming I alias AspNetUsers for AppUser) even though EF core doesn't use snake case by default or do I need to use snake case as an ef core naming convention through something like https://github.com/efcore/EFCore.NamingConventions

I'm trying to wrap my head around how to use both because I have a lot of code written already using Ormlite but the new Identity feature is extremely valuable as well.

**Comment by mythz** [comment, marked as answer]
OrmLite also uses snake_case  by default in its [PostgreSqlNamingStrategy.cs](https://github.com/ServiceStack/ServiceStack/blob/main/ServiceStack.OrmLite/src/ServiceStack.OrmLite.PostgreSQL/PostgreSqlNamingStrategy.cs) so it should be able to query the `AspNetUsers` table with a similar EF snake_case naming convention, however since it's using a different implementation there may be corner cases that differ.

Whenever it does differ you can use OrmLite's `[Alias]` attribute on Tables or columns to force it use a specific name, e.g:

```
[Alias("asp_net_users")]
public class AppUser : IdentityUser<int>
{
    [Alias("display_name")]
    public string DisplayName { get; set; }
}
```
